lib/msal-browser/src/crypto/CryptoOps.ts

```diff
@@ -26,23 +34,20 @@ export class CryptoOps implements ICrypto {
     private static EXTRACTABLE: boolean = true;
     private static POP_HASH_LENGTH = 43; // 256 bit digest / 6 bits per char = 43
 
+    private static DB_VERSION = 1;
+    private static DB_NAME = "msal.db";
+    private static TABLE_NAME =`${CryptoOps.DB_NAME}.keys`;
```

Comment:
These should be in constants

Reply:
These aren't used outside the context of the crypto APIs, so I chose to keep them here.